src/Orchard/Localization/Services/DefaultLocalizedStringManager.cs

```diff
@@ -254,20 +254,28 @@ public class DefaultLocalizedStringManager : ILocalizedStringManager {
             }
         }
 
+        private static string TrimQuote(string str) {
+            if (str.StartsWith("\"") && str.EndsWith("\"")) {

```

Comment:
Can you add a unit test just to be sure what you are fixing? I'd like to see for instance if it would make sense to to support  just one quote at the beginning or the end. Please see `src/Orchard.Tests/Localization`